nginx/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # a minimal Nginx container including containerbuddy and a simple virtulhost config
-FROM autopilotpattern/nginx:1-r6.1.0
+FROM autopilotpattern/nginx:branch-48-nginx_connections_unhandled_total-appears-incorrect
```

Comment:
This needs to be `autopilotpattern/nginx:1.13-r6.1.0` once https://github.com/autopilotpattern/nginx/pull/49 is merged